spec/models/chat_spec.rb

```diff
@@ -18,4 +18,37 @@
       expect(chat.reload.users).to match_array([user_a, user_b])
     end
   end
+
+  describe '#other_user' do
+    let(:user_a) { create(:user) }
+    let(:user_b) { create(:user) }
+    let(:chat) { Chat.create(user_a: user_a, user_b: user_b) }
+
+    it 'should return the other user' do
+      expect(chat.reload.other_user(user_b)).to eq(user_a)
```

Comment:
Mmm i always use a subject even though it contains just one test. I think makes the spec file easy to read...